Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ord"
 description = "◉ Ordinal wallet and block explorer"
-version = "0.10.0-ordit"
+version = "0.11.1"
```

Comment:
This change might overwrite our own custom changes right? @kodemon

Reply:
Should not have a negative impact on the functionality.